From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@tsubasaxZZZ__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (tsubasa.nomura). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

ktoliver commented:
@tsubasaxZZZ - Thanks for your contribution to the Azure docs. On November 16th, we announced the availability of our Azure technical documentation in a new location on [docs.microsoft.com](https://docs.microsoft.com). You can [read about it on our team blog](https://docs.microsoft.com/teamblog/docs-november-update). This change means the article source lives in a new GitHub repository. Because there are some changes in the source markdown as a result of the move, we can’t pull your contribution over to the new repo without creating a ton of merge conflicts. We invite you to resubmit the change (if you are willing) to the new version of the article at [https://github.com/Microsoft/azure-docs-pr/blob/master/includes/azure-storage-limits.md](https://github.com/Microsoft/azure-docs-pr/blob/master/includes/azure-storage-limits.md). Thanks for your understanding.

Commit:
.ActiveDirectoryInteractive mode in .NET 4.7.2

Commit:
Close, needs better Next Steps links. Now need MirekS to review.

Commit:
Applied feedback from PM MirekS (GithubMirek).

Commit:
Per MirekS review, added another screenshot (for app ID display), and revised some AD phrasing esp re MFA.

aaudu committed:
Updating DevEx docs for Build 2018

- Deprecate Windows Phone 8.x and Windows Store articles
- Redirect to root Azure AD Dev page

aaudu committed:
Fix Redirect Links

- Fix the redirect links with Mike's feedback, so they aren't pointing to dead locations

aaudu committed:
Fix Redirect Links

- Fix the redirect links with Mike's feedback, so they aren't pointing to dead locations

SaeedAkhter-MSFT committed:
388660 review active-directory-authentication-libraries

SaeedAkhter-MSFT committed:
Update active-directory-authentication-libraries.md

changing owner so feedback on the accuracy comes to our team

SaeedAkhter-MSFT committed:
388661 review v2 lib doc

SaeedAkhter-MSFT committed:
acrolinx issues

aaudu committed:
Fix references to deprecated docs

aaudu committed:
Merge branch 'master' into aaudu-build-2018-patches-devex

sromeroz committed:
Adding article on B2C audit logs

aaudu committed:
Merge branch 'master' into aaudu-build-2018-patches-devex

chrissie926 committed:
iot hub module articles

chrissie926 committed:
address dominic's comments

chrissie926 committed:
address dominic's comments

aaudu committed:
Merge branch 'master' into aaudu-build-2018-patches-devex

chrissie926 committed:
add artiles

chrissie926 committed:
fix build issues

v-viinde committed:
USER STORY 1246769 SaaS App Tutorial: QuickHelp Update

chrissie926 committed:
fix dominic's comments

chrissie926 committed:
add new article

v-viinde committed:
Updated

iainfoulds committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

chrissie926 committed:
fix build issue

iainfoulds committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

DCtheGeek committed:
Adding schema for GH Issue #7829

DCtheGeek committed:
Fixing typo

DCtheGeek committed:
Removing ticks since they didn't render

aaudu committed:
Merge branch 'master' into aaudu-build-2018-patches-devex
Fix: Deprecate Windows Phone and Store articles

JasonWHowell committed:
New article for threshold based alerts

JasonWHowell committed:
Acrolinx

JasonWHowell committed:
Acrolinx

JasonWHowell committed:
Acrolinx

JasonWHowell committed:
Acrolinx

JasonWHowell committed:
Acrolinx

linda33wj committed:
Fix Web table and add properties for Azure MySQL & PG articles

Commit:
Updated with changes from jinyam.

redirection.json: added redirect from index.md to Home (Overview).
cognitive-services/index.yml: made main CS page link to Home.

Home: renamed to Home-v1 and added important/link to new Home/APIv2.
Added new Home for v2 and added note/link to Home-v1/APIv1.

toc: made Overview a dropdown with 1 item named "What is Computer Vision".
Changed "How to" to "How to guides".
Added new APIv2 to Reference.

Commit:
Deleted index.
Moved Quickstarts and Tutorials under How-to and renamed.

Commit:
Add back in. index no longer in TOC.

SaeedAkhter-MSFT committed:
address feedabck

Sammak committed:
Updates for Build

Includes age gating, b2clogin, new policy types, and updates for password complexity and language customization

CelesteDG committed:
Updated author and date metadata

Update author, ms.author, and ms.date metadata values.

Sammak committed:
TOC fixes and style

Sammak committed:
Added to known issues

aaudu committed:
Merge pull request #1 from CelesteDG/patch-9

Updated author and date metadata

v-deken committed:
Updated overviews for News Search, Video Search, and Web Search

v-nagta committed:
User Story 1248717: SaaS App Tutorial: Manabi Pocket

v-deken committed:
Updated Web Search overview to fix heading depth

Commit:
.

v-nagta committed:
update

v-nagta committed:
update

rothja committed:
Fixing screenshots

rothja committed:
Unsaved change added

iainfoulds committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

JasonWHowell committed:
Update stream-analytics-manage-job.md

JasonWHowell committed:
Updating date

sromeroz committed:
Added screenshots and minor formatting changes

sromeroz committed:
Minor styling changes

sromeroz committed:
Fixed link to external article

rolyon committed:
Changes for experiment 50f26fb6-3aa0-42

rolyon committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into rolyon-aad-landing-experiment

douglaslMS committed:
Made PM fixes for recently discovered automation issue.

raiye committed:
Update virtual-machines-common-how-to-enable-write-accelerator.md

added portal enablement

Commit:
caleb06

CelesteDG committed:
Table formatting resolution and copyedits

Updated to resolve issue with the formatting on the table, made all tables consistent (to have a line after the last entry), and suggested some copyedits.

SaeedAkhter-MSFT committed:
acrolix

roygara committed:
Adding image.

roygara committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into addImage

Commit:
caleb06

BryanLa committed:
tweaks for Tim

voellm committed:
Update database-security.md

Updating client to server settings.

BryanLa committed:
tweaks for Tim

parakhj committed:
Minor updates to articles and TOC

raiye committed:
Merge pull request #5 from roygara/addImage

Add image

raiye committed:
Update virtual-machines-common-how-to-enable-write-accelerator.md

parakhj committed:
Removing support for MSA accounts

parakhj committed:
Minor updates

BryanLa committed:
add IOT Suite PCS to Quickstart list

Commit:
Removed duplicate entry for the vscode article

This duplicate was in the getting-started section - it shouldn't be in that section since it is not a getting started article

Commit:
Update TOC.md

Commit:
Create data-lake-analytics-policies.md

v-jasowells committed:
Merge branch 'master' into TranslatorV3

Commit:
Fixed broken link

CelesteDG committed:
Added metadata for include file

Added metadata for the include file to address PRMerger comment and unblock the PR.

v-deken committed:
Updated overviews for Autosuggest, Custom Search, Entity Search, and Image Search

twooley committed:
Updated next steps

douglaslMS committed:
Starting PM updates.

sethmanheim committed:
Updates to premium messaging info

aaudu committed:
Merge pull request #3 from CelesteDG/patch-12

Added metadata for include file

aaudu committed:
Merge pull request #2 from CelesteDG/patch-10

Table formatting resolution and copyedits

chrissie926 committed:
address review comments

v-deken committed:
Updated overviews for Video Search

jlian committed:
Add placeholder GUIDs for clarity

chrissie926 committed:
fix comments

Commit:
QnA 2

mmacy committed:
[ACR] geo-rep screenshot update

jlian committed:
Fix typo

MS-Santi committed:
Metadata and Link changes as requested by ktoliver

Changes required to be able to merge

Commit:
Merge pull request #39529 from JasonWHowell/patch-2

Update stream-analytics-manage-job.md

Commit:
Merge pull request #39621 from jlian/guid-placeholder

Add placeholder GUIDs for clarity

douglaslMS committed:
Added new section about recreating alerts.

ktoliver committed:
Merge pull request #39143 from v-viinde/quickhelpupdate

USER STORY 1246769 SaaS App Tutorial: QuickHelp Update

Sammak committed:
Added instructions for using b2clogin.com

iainfoulds committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

ktoliver committed:
fix metadata formatting

edmacauley committed:
removing preview language

iainfoulds committed:
[VMSS] Edits and revisions for consistency

ktoliver committed:
Merge pull request #37798 from aaudu/aaudu-build-2018-patches-devex

Deprecate Windows Phone 8.x and Windows Store articles

ktoliver committed:
Merge pull request #38921 from chrissie926/release-build-module-twin

iot hub module articles

hirokib committed:
fixing misspelling 

parallelized not paralyze

iainfoulds committed:
[VM] Updating author metadata

iainfoulds committed:
Update extensions-nwa.md

TrishaMc committed:
Merge pull request #39505 from v-nagta/manabipocket

User Story 1248717: SaaS App Tutorial: Manabi Pocket

v-shils committed:
Merge pull request #39553 from douglaslMS/adf-everest-fix1

Made PM fixes for recently discovered automation issue.

ktoliver committed:
Merge pull request #39496 from Sammak/master

Updates for Build

tfosmark committed:
Merge pull request #39504 from v-deken/Overviews-Search

Updated overviews for News Search, Video Search, and Web Search

markingmyname committed:
[PBIE] Edited Get Available Feature info

markingmyname committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into APBIE

ktoliver committed:
Merge pull request #39554 from rolyon/rolyon-aad-landing-experiment

[Azure AD] Data validation experiment 50f26fb6-3aa0-42

Commit:
Fixing build issues.

TrishaMc committed:
Merge pull request #39562 from raiye/patch-2

Update virtual-machines-common-how-to-enable-write-accelerator.md

Commit:
Fixing build issues.

v-shils committed:
Merge pull request #39565 from MarkusVi/caleb06

caleb06

tfosmark committed:
Merge pull request #39573 from BryanLa/bryanla-iot-lp-tweaks

Bryanla iot lp tweaks

garyericson committed:
TDSP metadata update(1): changing author from bradsev to deguhath

v-shils committed:
Merge pull request #39574 from parakhj/master

Minor updates to articles and TOC

garyericson committed:
TDSP metadata update(2): changing author from bradsev to deguhath

garyericson committed:
TDSP metadata update(3): changing author from bradsev to deguhath

v-shils committed:
Merge pull request #39605 from twooley/master

Updated next steps

garyericson committed:
TDSP metadata update(4): changing author from bradsev to deguhath

garyericson committed:
TDSP metadata update(5): changing author from bradsev to deguhath

v-shils committed:
Merge pull request #39606 from sethmanheim/prem-fix

Updates to premium messaging info

garyericson committed:
TDSP metadata update(6): changing author from bradsev to deguhath

garyericson committed:
TDSP metadata update(7): changing author from bradsev to deguhath

garyericson committed:
TDSP metadata update(8): changing author from bradsev to deguhath

v-shils committed:
Merge pull request #39625 from mmacy/geo-rep-tut-images

[ACR] geo-rep screenshot update

elisolMS committed:
removed unnecessary section

garyericson committed:
AML metadata clean-up

Commit:
Merge pull request #39651 from garyericson/04-30-tdsp-author

TDSP metadata update(1): changing author from bradsev to deguhath

Commit:
Merge pull request #39659 from elisolMS/update-section

removed unnecessary section

rmca14 committed:
Merge pull request #36584 from MightyPen/adi-20180329-a

.ActiveDirectoryInteractive mode in .NET 4.7.2

tfosmark committed:
Merge pull request #39506 from v-jaswel/TranslatorV3

Replacement for PR 36493

ktoliver committed:
Merge pull request #39633 from douglaslMS/adf-v1-alerts

 Added new section about recreating alerts for v1.

Commit:
minor fixes

PRMerger13 committed:
Merge pull request #39652 from garyericson/04-30-02-tdsp-author

TDSP metadata update(2): changing author from bradsev to deguhath

garyericson committed:
Minor metadata clean-up (typo)

sromeroz committed:
Merge remote-tracking branch 'upstream/master' into b2c-reference-audit-logs

PRMerger15 committed:
Merge pull request #39643 from hirokib/patch-7

fixing misspelling

PRMerger16 committed:
Merge pull request #39665 from dkkapur/master

minor fixes

PRMerger17 committed:
Merge pull request #39653 from garyericson/04-30-03-tdsp-author

TDSP metadata update(3): changing author from bradsev to deguhath

PRMerger17 committed:
Merge pull request #39654 from garyericson/04-30-04-tdsp-author

TDSP metadata update(4): changing author from bradsev to deguhath

garyericson committed:
Metadata clean-up(1): fixing author values

PRMerger17 committed:
Merge pull request #39666 from garyericson/04-30-cogserv-metadata

Minor metadata clean-up (typo)

PRMerger17 committed:
Merge pull request #39655 from garyericson/04-30-05-tdsp-author

TDSP metadata update(5): changing author from bradsev to deguhath

sromeroz committed:
Added article to TOC

PRMerger18 committed:
Merge pull request #39656 from garyericson/04-30-06-tdsp-author

TDSP metadata update(6): changing author from bradsev to deguhath

PRMerger18 committed:
Merge pull request #39657 from garyericson/04-30-07-tdsp-author

TDSP metadata update(7): changing author from bradsev to deguhath

PRMerger18 committed:
Merge pull request #39658 from garyericson/04-30-08-tdsp-author

TDSP metadata update(8): changing author from bradsev to deguhath

MS-Santi committed:
folder structure

MS-Santi committed:
images for DR and geo-distribution article

garyericson committed:
Metadata clean-up(2): removing multiple authors

MS-Santi committed:
removed placeholder

MS-Santi committed:
Update TOC.yml

MS-Santi committed:
Update TOC.yml

MS-Santi committed:
Added DR and Geo-Distribution to TOC.yml

chrissie926 committed:
fix small issues

Commit:
Update text-analytics-supported-languages.md

Commit:
Update data-lake-analytics-policies.md

Commit:
controls02

Commit:
controls02

Commit:
conditions05

GitHubber17 committed:
Merge pull request #39590 from saveenr/patch-34

Removed duplicate entry for the vscode article

PRMerger14 committed:
Merge pull request #39692 from MarkusVi/conditions05

conditions05

GitHubber17 committed:
Merge pull request #39540 from sromeroz/b2c-reference-audit-logs

B2C reference audit logs

GitHubber17 committed:
Merge pull request #39596 from saveenr/patch-37

Fixed broken link

GitHubber17 committed:
Merge pull request #39689 from MarkusVi/controls02

controls02

Commit:
Update data-lake-analytics-policies.md

PRMerger17 committed:
Merge pull request #39660 from garyericson/04-30-ml-metadata-cleanup

AML metadata clean-up

gsilva5 committed:
Updating Linux, 2 core VMs, enable on existing

Linux update to AN docs to outline how to enable on existing VMs and resize.  Few additional improvements.

daveba committed:
updates based on PM feedback

daveba committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into update-arm-template-docs

daveba committed:
updates based on PM feedback

GitHubber17 committed:
Merge pull request #39593 from saveenr/patch-36

Add doc explaining for to use policies for Azure Data Lake

Commit:
Removing unused files.

Commit:
Image replacement.

MRayermannMSFT committed:
Add 1.0.0 rel notes

SaeedAkhter-MSFT committed:
address feedback

Commit:
Merge pull request #39705 from MRayermannMSFT/dev/marayerm/1.0.0-rel-notes

Add 1.0.0 rel notes for Storage Explorer

JasonWHowell committed:
fixing timestamps

JasonWHowell committed:
fixing timestamps

JasonWHowell committed:
fixing timestamps

KumudD committed:
added sample and toc restructure

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into dnssamplescripts0430

KumudD committed:
fixed typo

mrbullwinkle committed:
pricing updates from dale 04_30_2018

mrbullwinkle committed:
fix quotes

PRMerger17 committed:
Merge pull request #39712 from mrbullwinkle/mrb_05_01_2018_pricing_updates

pricing updates from dale 04_30_2018

Commit:
Temp removal of ComputerVision redirect for PR warning.

v-deken committed:
Reordered Resources section in TOC for all Search services to include pricing & match standards

PRMerger13 committed:
Merge pull request #39572 from voellm/patch-1

Update database-security.md

PRMerger13 committed:
Merge pull request #39648 from markingmyname/APBIE

[PBIE] Edit Get Available Feature info

rothja committed:
Fixing PR blocking issues

Commit:
freshness

PRMerger20 committed:
Merge pull request #39408 from DCtheGeek/dmc-policy-patch2

Adding schema for GH Issue #7829

Commit:
Merge pull request #39673 from garyericson/04-30-01-cogserv-fix-author

Metadata clean-up(1): fixing author values

itechedit committed:
Merge pull request #37871 from SaeedAkhter-MSFT/saeeda-aad-auth-libs

388660 review active-directory-authentication-libraries

rjagiewich committed:
Merge pull request #37876 from SaeedAkhter-MSFT/saeeda-aad-libs-v2

388661 review v2 lib doc

Commit:
Merge pull request #39717 from Blackmist/freshness

freshness

TrishaMc committed:
Merge pull request #39422 from JasonWHowell/sa-threshold

New article for threshold based alerts

KumudD committed:
toc update

KumudD committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into dnssamplescripts0430

Commit:
Remove note

ktoliver committed:
Merge pull request #39463 from easyj2j/ComputerVision

Updated with changes from jinyam

PRMerger16 committed:
Merge pull request #39721 from Jann-Skotdal/master

Remove prerelease note

cwatson-cat committed:
Updated meta-data for a few billing topics

eross-msft committed:
adding hybrid include

gsilva5 committed:
Linux AN, incorporating jdial comments

rjagiewich committed:
Merge pull request #39551 from rothja/akvupdates2

Updating screenshots with help from CSS for AKV SQL VM article

ktoliver committed:
Merge pull request #39669 from v-jaswel/QnA-pr-2

QnA PR 2 of 2

rjagiewich committed:
Merge pull request #39677 from MS-Santi/MS-Santi-DR-and-GeoDistr

Pull request for DR and Geo-Distribution

Commit:
Merge pull request #39645 from iainfoulds/patch-4

[VM] Updating author metadata

douglaslMS committed:
Fixed two typos caught by previous PR review.

Commit:
Merge pull request #39726 from cwatson-cat/5-1-billing-metadataupd

Updated meta-data for a few billing topics

rjagiewich committed:
Merge pull request #39697 from daveba/update-arm-template-docs

[Azure AD MSI] Update arm template docs

Commit:
Update metadata.

PRMerger10 committed:
Merge pull request #39441 from linda33wj/master

Fix Web table and add properties for Azure MySQL & PG articles

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into ComputerVision

PRMerger12 committed:
Merge pull request #39732 from douglaslMS/adf-fix-typos

Fixed two typos caught by previous PR review.

iainfoulds committed:
[VMSS} Minor review edits

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

TrishaMc committed:
Merge pull request #39709 from KumudD/dnssamplescripts0430

DNS Sample scripts

rmca14 committed:
Merge pull request #39715 from v-deken/Resources-Search

Reordered Resources section in TOC for all Search services to include…

PRMerger17 committed:
Merge pull request #39674 from garyericson/04-30-02-cogserv-fix-author

Metadata clean-up(2): removing multiple authors

v-alje committed:
Merge pull request #39739 from MicrosoftDocs/FromPublicRepo

From public repo

PRMerger16 committed:
Merge pull request #39678 from chrissie926/release-build-module-twin

fix small issues

ktoliver committed:
Merge pull request #39638 from iainfoulds/vmssencryptionedits

[VMSS] Scale set disk encryption

rmca14 committed:
Merge pull request #39729 from eross-msft/lizross-include-creation

Created the new Include file for GDPR and hybrid scenarios

v-shils committed:
Merge pull request #39637 from edmacauley/localworking

removing preview language

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

ktoliver committed:
Merge pull request #39695 from gsilva5/patch-17

Updating Linux, 2 core VMs, enable on existing

v-alje committed:
Merge pull request #39771 from MicrosoftDocs/FromPublicRepo

From public repo

ktoliver committed:
Merge pull request #39736 from easyj2j/ComputerVision

Computer Vision - metadata

GitHubber17 committed:
Merge pull request #39681 from ashmaka/patch-2

Adding Entity Linking to Supported Languages
